lib/error.js

```diff
@@ -105,13 +105,7 @@ for (var status in statusCodes) {
 
   Util.inherits(error, exports.HttpError)
 
-  var className = toCamelCase(defaultMsg)
+  var className = defaultMsg.replace(/\s/g, '')
```

Comment:
This looks like it will change functionality. What are the ramifications?

Reply:
In the context of this file, the `toCamelCase` method just turned messages like `Requested Range Not Satisfiable` into an Error name like `RequestedRangeNotSatisfiable`. It used its own toCamelCase method, not the Util.toCamelCase one

https://github.com/octokit/node-github/blob/fb0ad883f6f79007d41f6a94258c9b8a09fc2454/lib/error.js#L113-L117

I've double checked that, it's all good